.github/workflows/lychee-push.yml

```diff
@@ -0,0 +1,13 @@
+on: push
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: lychee Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.0.6
+        with:
+          args: --exclude ".*.signalfx.com" --exclude "splunk.jfrog.io.*basearch" --exclude "localhost:.*" -v -n "*.md" "**/*.md"
+      - name: Fail if there were link errors
+        run: exit ${{ steps.lychee.outputs.exit_code }}
```

Comment:
I thought if the step exits non-zero it fails the action? https://docs.github.com/en/actions/creating-actions/setting-exit-codes-for-actions

Reply:
It may -- this is from the markplace readme. Not sure why they recommend....